src/vellum/utils/templating/render.py

```diff
@@ -35,6 +56,8 @@ def render_sandboxed_jinja_template(
         if jinja_custom_filters:
             environment.filters.update(jinja_custom_filters)
 
+        environment.filters["replace"] = custom_replace
```

Comment:
[blocking] let's move this above `jinja_custom_filters` to let the user provide their own if so inclined